### PR TITLE
Make icons accessible to screen readers

### PR DESCRIPTION
**Pull request by maximevaillancourt**
**What's the goal of this PR?**
This PR adds a `role="img"` attribute and a valid `<title>` tag to each payment icon SVG matching the value in the `payment_icons.yml` file. This way, screen readers can read out the icon names to visually impaired users. 🎉 

This PR also adds tests to make sure every SVG file meets the above accessibility requirements.

**Question to reviewers**
Should the icons be optimized with SVGO before merging? Looks like my script (using Nokogiri) added carriage returns to the files. I could also rerun the script with `document.to_xml(:save_with => Nokogiri::XML::Node::SaveOptions::AS_XML)`, which should remove the newlines.

Sorry for the size of this PR (all SVGs were updated). The only 'non-SVG' changes are the new tests, which you can see in [this file](https://github.com/activemerchant/payment_icons/pull/87/files#diff-8dff92b1651886640f3464753fb91848R3).

### CODE REVIEW

**Comment by maximevaillancourt** [comment]
cc/ @joshnuss @nathanpjf
Forgot to add you as a reviewer when creating the PR.

File: test/unit/payment_icon_test.rb [hunk 1]
```diff
@@ -14,4 +16,40 @@ class PaymentIconTest < ActiveSupport::TestCase
...
+    PaymentIcon.all.each do |icon|
+      assert File.exist?(ICONS_DIRECTORY.join("#{icon.name}.svg"))
+    end
+  end
+
+  test 'Every payment icon SVG has a valid XML tree' do
+    PaymentIcon.all.each do |icon|
+      file = File.read(ICONS_DIRECTORY.join("#{icon.name}.svg")).freeze
+      document = Nokogiri::XML.parse(file)
+      assert document.errors.blank?, message: "The '#{icon.name}' SVG file is invalid: #{document.errors}"
```
Comment (joshnuss) [inline test/unit/payment_icon_test.rb]: Shall we recommend using SVGO in the error message?

Comment (joshnuss) [reply test/unit/payment_icon_test.rb]: > Does SVGO provide suggestions to fix an invalid SVG document?
Not sure actually 😊

For "\n" characters, we don't want them cause of performance right?
If so, then maybe we shouldn't allow whitespace either, which could be done with `/(^\s)|(\s$)/`

Comment (maximevaillancourt) [reply test/unit/payment_icon_test.rb]: Come to think of it, this sort of optimization might be a little overkill for our needs. I'll mention SVGO in the error message, and combined with the PR checklist containing the SVGO bit, we should be good. :+1:

**Comment by maximevaillancourt** [comment]
I optimized all SVGs using SVGO, and added a mention to SVGO in the error message. :+1:

**Comment by joshnuss** [comment]
Thanks @maximevaillancourt!

**Pull request merged**